content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,400 @@
+and use the web-based graphical user interface to view service graphs of
+the mesh and your Istio configuration objects. Lastly, you use the Kiali
+Public API to generate graph data in the form of consumable JSON.
+
+The [Bookinfo](/docs/examples/bookinfo/) sample application is used as the
+example application throughout this task.
+
+## Before you begin
+
+Create a secret in your Istio namespace with credentials that you use to
+authenticate to Kiali. See the
+[Helm README](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/README.md#installing-the-chart)
+for details, but here is an example command you can modify and run to create a secret:
+
+```bash
+USERNAME=$(echo -n 'admin' | base64)
+PASSPHRASE=$(echo -n 'mysecret' | base64)
+NAMESPACE=istio-system
+kubectl create namespace $NAMESPACE
+cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kiali
+  namespace: $NAMESPACE
+  labels:
+    app: kiali
+type: Opaque
+data:
+  username: $USERNAME
+  passphrase: $PASSPHRASE
+EOF
+```
+
+Once the Kiali secret is created, install Kiali via Helm by following
+[these instructions](/docs/setup/kubernetes/helm-install/). Note that you need
+to use the `--set kiali.enabled=true` option when running the `helm` command.
+
+For example:
+
+{{< text bash >}}
+$ helm template --set kiali.enabled=true install/kubernetes/helm/istio --name istio --namespace istio-system > $HOME/istio.yaml
+$ kubectl apply -f $HOME/istio.yaml
+{{< /text >}}
+
+> {{< idea_icon >}} This Task does not discuss Jaeger and Grafana; however, if
+you have them already installed in your cluster and you want to see how Kiali
+integrates with both of them, you need to pass additional arguments to the
+`helm` command. For example:
+
+{{< text bash >}}
+$ helm template \
+    --set kiali.enabled=true \
+    --set "kiali.dashboard.jaegerURL=http://$(kubectl get svc tracing -o jsonpath='{.spec.clusterIP}'):80" \
+    --set "kiali.dashboard.grafanaURL=http://$(kubectl get svc grafana -o jsonpath='{.spec.clusterIP}'):3000" \
+    install/kubernetes/helm/istio \
+    --name istio --namespace istio-system > $HOME/istio.yaml
+$ kubectl apply -f $HOME/istio.yaml
+{{< /text >}}
+
+Once Istio and Kiali are installed, deploy the [Bookinfo](/docs/examples/bookinfo/) sample application.
+
+> {{< idea_icon >}} The above installation instructions assume you have `helm`
+and want to use it to install Kiali. Alternatively, you can install Kiali by
+following the [Kiali install instructions](https://www.kiali.io/gettingstarted/).
+
+## Generating a service graph
+
+1.  Verify that the service is running in your cluster.
+
+    In Kubernetes environments, execute the following command:
+
+    {{< text bash >}}
+    $ kubectl -n istio-system get svc kiali
+    NAME           CLUSTER-IP      EXTERNAL-IP   PORT(S)    AGE
+    kiali          10.59.253.165   <none>        20001/TCP  30s
+    {{< /text >}}
+
+1.  Determine the Bookinfo URL.
```

Comment:
done - however, I used the term "GATEWAY_URL" since that is the term/variable name throughout. To avoid confusing the reader, and to be clear we are talking about the env var that needs to be set, I felt it best to use GATEWAY_URL always (rather then here say "gateway URL").